projects/core/src/cms/store/effects/navigation-entry-item.effect.ts

```diff
@@ -52,10 +52,10 @@ export class NavigationEntryItemEffects {
                 )
             )
           );
-        } else if (data.ids.pageIds.length > 0) {
+          //} else if (data.ids.pageIds.length > 0) {
           // TODO: future work
           // dispatch action to load cms page one by one
-        } else if (data.ids.mediaIds.length > 0) {
+          //} else if (data.ids.mediaIds.length > 0) {
```

Comment:
Perhaps the dead code should be removed.

Reply:
navigation entry can also point to medias and pages. Now, we only handle links. So, this dead code is a reminder for future work.